hooks/publish_extra_details.py

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2015 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+from sgtk import Hook
+
+
+class PublishExtraDetails(Hook):
+    """Hook that can be used to add extra fields in the details pane."""
+
+    def execute(self, sg_item, **kwargs):
+        """
+        Main hook entry point.
+
+        :param sg_item: published item
```

Comment:
I'd call this something other than "sg_item". Maybe something like the following?

```
:param entity: The entity dictionary associated with the published item to be displayed in the loader app.
```